CountItems/utils/index.js

```diff
@@ -23,19 +23,34 @@ function getIsLocationTransientCb(log, locationConfigFile) {
     };
 }
 
+function getMongoDbConfig(log, configFile) {
+    let config;
+    try {
+        const data = fs.readFileSync(configFile, { encoding: 'utf-8' });
+        config = JSON.parse(data).mongodb;
+    } catch (err) {
+        log.info(`could not parse config file: ${err.message}`);
```

Comment:
Maybe `warn` here? As using a config file should be the preferred way of running this function (according to your code).

Reply:
Even `error`, you can't parse your config file you should fail. in my opinion.